monero/wallet_client.go

```diff
@@ -43,20 +43,20 @@ const (
 // WalletClient represents a monero-wallet-rpc client.
 type WalletClient interface {
 	GetAccounts() (*wallet.GetAccountsResponse, error)
-	GetAddress(idx uint64) (*wallet.GetAddressResponse, error)
+	GetAddress(idx uint32) (*wallet.GetAddressResponse, error)
```

Comment:
I did [a PR](https://github.com/MarinX/monerorpc/pull/18) on MarinX's repo to fix some features we don't use for the atomic swaps and adjusted these to match what `monero-wallet-rpc` is using to represent them on the server side. Matching them here too in order to update to the latest release.